### ISSUE TITLE
LIST_RANGE on a VAR doesn't limit to the VAR

### ISSUE THREAD

**Issue by joningold** [inkle/ink#442]
```
LIST abc = a, b, c, d, e, f, g, h, i 
VAR vowels = (a, e, i )

{LIST_RANGE(vowels, 1, 5)} 
```

Results in "a,b,c,d,e" not "a,e". I'm not sure that's wrong, but it's not what I expected.

Note that {LIST_RANGE(abc, 1, 5)} gives the same result, and abc is "empty", so presumably LIST_RANGE is intended to "create from the definition" rather than "filter from the current".

**Comment by joethephish** [comment]
Yeah, this was designed to be used with a source definition, but I allowed it to work with var lists for convenience. We could always repurpose it though, you you’d just have to do `LIST_RANGE(LIST_ALL(...))` to get the old behaviour.

**Comment by joningold** [comment]
yeah (i think i do the LIST_ALL thing habitually anyway!)

**Comment by joethephish** [comment]
Fixed in 4098dcc25e221c12f88a1d6a022070448fe3b9af.
